src/renderer/helpers/logger.test.ts

```
@@ -0,0 +1,63 @@
+
+      logger.error('e')
+      logger.warn('w')
+      logger.info('i')
+      logger.debug('d')
+
+      expect(spies.error).toHaveBeenCalledOnce()
+      expect(spies.warn).toHaveBeenCalledOnce()
+      expect(spies.info).toHaveBeenCalledOnce()
+      expect(spies.debug).toHaveBeenCalledOnce()
+
+      Object.values(spies).forEach((s) => s.mockRestore())
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)